### ISSUE TITLE
Cell and big content

### ISSUE THREAD

**Issue by Frulenzo** [dhorions/boxable#42]
Unfortunately when cell has big content (bigger height than next page size), cell isn't properly formated through multiple pages.

Example
[cellWithBigContent.pdf](https://github.com/dhorions/boxable/files/202458/cellWithBigContent.pdf)

**Comment by dhorions** [comment]
This was also fixed by #49  , right?

**Comment by Frulenzo** [comment]
Not really. It solved only "missing" top border if your table is big but when your cell height is bigger than your canvas height than we have problem. We need to implement in a way that we enable "breaking" cell (cell that can be across multiple pages).

In your test cases just give a high cell height, something like :

```java
 Row<PDPage> row2 = table.createRow(800f);
```

**Comment by ExLeonem** [comment]
I'm working on a commercial project that just recently started using boxable. The formatting of HTML elements within cells as well as page breaks within cells are a requirement for us. 

We tried to fix this bug in our code by using the method `Paragraph#getLines` to get the lines the library is trying to draw for a single cell and distribute them to multiple cells.

In it's current state the `Paragraph#getLines` method does not preserve HTML tags. So we added a flag to preserver them. Another issue that we encountered was that distributing the lines to multiple cells may destroy the HTML formatting. So there needs to be a mechanism to add unclosed or missing open HTML tags to the beginning/end of the distributed contents. 

I could provide a PR with a slightly refactored `Paragraph#getLines` + an added flag to preserve the HTML Tags.

Are there any plans to fix this bug in the near future? Because we ideally need this to be included in the library instead
of manual fixing this in our own code. #64 seems to be a similiar issue.

**Comment by quodlibetbv** [comment]
Already resolved, see unit test src\test\java\be\quodlibet\boxable\CellOverflowAcrossPagesTest.java

[CellOverflowAcrossPagesTest.pdf](https://github.com/user-attachments/files/25151641/CellOverflowAcrossPagesTest.pdf)
